### DISCUSSION TITLE
Sample app that sends a midi note

### DISCUSSION THREAD

**Discussion by svongripp** [InvisibleWrench/FlutterMidiCommand#121] [General]
Wondering if someone here happens to have a very simple sample app they could share that sends a midi note using this package? I'm wanting to explore this basic use case. Thanks so much.

**Comment by mortenboye** [comment]
The example app, shows how to do this.

The process in short:
- Discover available MIDI devices.
- Establish connection to a specific device.
- Send and receive MIDI data.
